sdk/wallet-adapter/example/src/App.tsx

```diff
@@ -12,9 +12,9 @@ transaction.add(
   Commands.MoveCall({
     target: `0x2::devnet_nft::mint`,
     arguments: [
-      transaction.input("foo"),
-      transaction.input("bar"),
-      transaction.input("baz"),
+      transaction.pure("foo"),
+      transaction.pure("bar"),
+      transaction.pure("baz"),
     ],
```

Comment:
@Jordan-Mysten , should we make this a part of the turbo build?

Reply:
Yeah we're planning on rebuilding the demo app entirely so I'll probably do it with that